postgres-appliance/Dockerfile

```diff
@@ -44,12 +44,12 @@ RUN curl -L https://github.com/coreos/etcd/releases/download/v${ETCDVERSION}/etc
 
 ENV PATH $PATH:/usr/lib/postgresql-${PGVERSION}/bin
 
+# Copy the snakeoil certificates for usage as dummy certificates
+RUN cp /etc/ssl/private/ssl-cert-snakeoil.key /home/postgres/dummy.key && cp /etc/ssl/certs/ssl-cert-snakeoil.pem /home/postgres/dummy.crt
```

Comment:
It is not owned by the postgres user, but is from root:ssl-cert
During runtime it cannot access the real file.

What we could do is add postgres to ssl-cert group, and not even copy it.